decidim-core/app/helpers/decidim/icon_helper.rb

```diff
@@ -15,17 +15,36 @@ module IconHelper
       "Decidim::Blogs::Post" => "pen-nib-line",
```

Comment:
this should be better represented.

Reply:
:thinking: what's your point?

Reply:
I am just thinking this needs to be refactored ( maybe by maintainers), in order to decidim core module agostic ( like decidim core sould not know about Blogs )

Reply:
Created an issue for maintainers to review